front/src/modules/ui/table/components/EntityTableCell.tsx

```diff
@@ -10,6 +11,14 @@ import { ColumnIndexContext } from '../contexts/ColumnIndexContext';
 import { GenericEditableCell } from '../editable-cell/components/GenericEditableCell';
 import { useCurrentRowSelected } from '../hooks/useCurrentRowSelected';
 
+const StyledCell = styled.div`
+  &:hover {
```

Comment:
Let's not introduce a new container, we could make this happen in EditableCell directly as we are already tracking the hover state. 
We can pass isHovered state from `EditableCell` to `EditableCellDisplay` (and `EditableCellDisplayContainer`) as prop. then in `EditableCellDisplayContainer` use it to display the same style as for `softFocus` case in `StyledEditableCellDisplayModeOuterContainer`

Reply:
Thanks!